falcon_apispec/falcon_plugin.py

```diff
@@ -28,6 +35,8 @@ def _generate_resource_uri_mapping(app):
                 for method_name, method_handler in route.method_map.items():
                     if method_handler.__dict__.get("__module__") == "falcon.responders":
```

Comment:
This one needs to be fixed as well, because I'll not work with `falcon==3.0`, neither.

```suggestion
                    if method_handler.__module__ == "falcon.responders":
```

Reply:
Hey @aslantar I'm  using this awesome plugin, but I really would like to use it with falcon3, can I send these changes again in another repo?, I've tested and seems it is working fine with falcon3, what do you think?

Reply:
Sorry @aslantar, I've been busy lately so I missed your review.
Thanks @Javlopez, please help me fix this problem, I don't have enough time to continue to maintain this PR

Reply:
@Javlopez, sure. I'd love to see the changes integrated.

Reply:
Hey  @aslantar the PR is here https://github.com/alysivji/falcon-apispec/pull/34 please take a look

Reply:
@Javlopez why do you need a separate PR with the same changes, instead of assisting @vuonglv1612 to get this one done?

Reply:
hello, sorry I didn't see this before, well just I found a way to send eveything in one PR, but you are right I can send my changes to @vuonglv1612 repo, for me is ok, I gonna send them today later

Reply:
Hey @aslantar I've opened a PR for in the fork https://github.com/vuonglv1612/falcon-apispec/pull/1 just with the missing changes,  @vuonglv1612  can you take a look please?

Best Regards